ote/src/cljs/ote/views/admin/sea_routes.cljs

```diff
@@ -64,7 +64,9 @@
                                (if (true? ((weekday-index-to-keyword last-weekday-index) route))
                                  (* -1 index)
                                  (recur (inc index) (dec last-weekday-index)))
-                               nil))]
+                               ; return 0 days only if to-date and weekday are given.
```

Comment:
```suggestion
                               ;; return 0 days only if to-date and weekday are given.
```
https://guide.clojure.style/#two-semicolons-for-code-fragment

+Add a "why" into the comment if reasonable.

Reply:
why added